tools/test/audio/sof_test_perf.m

```diff
@@ -182,6 +184,28 @@
 
 end
 
+%% Reference: AES17 6.4.1 Dynamic range
+function [fail, dr_db] = dr_test(t)
+
+test = test_defaults(t);
+
+% Create input file
+test = dr_test_input(test);
+
+% Run test
+% test = test_run_process(test, t);
+test = remote_test_run(test);
+
+% Measure
+test = dr_test_measure(test);
```

Comment:
possible have more var name? instead of use test always, it may make confused or generate unexpected result, what do you think?